examples/target_group_only/.happy/terraform/envs/rdev/main.tf

```diff
@@ -0,0 +1,97 @@
+module "stack" {
+  source = "git@github.com:chanzuckerberg/happy//terraform/modules/happy-stack-eks?ref=heathj/target-group-only"
+
+  image_tag        = var.image_tag
+  image_tags       = jsondecode(var.image_tags)
+  stack_name       = var.stack_name
+  deployment_stage = "rdev"
+  stack_prefix     = "/${var.stack_name}"
+  k8s_namespace    = var.k8s_namespace
+
+  services = {
```

Comment:
@jgadling this is how you wire up your existing ALB

Reply:
this looks great!!